### DISCUSSION TITLE
Who is using Apache Kyuubi？

### DISCUSSION THREAD

**Discussion by yaooqinn** [apache/kyuubi#925] [Show and tell]
## Who's using/谁在使用 Apache Kyuubi?

![kyuubi_logo (2)](https://user-images.githubusercontent.com/8326978/129223344-853d1268-6b8a-4113-abde-01a23b4b074c.png)

Travel alone is fast. Travel together will take you far.
独行快，众行远。

The Apache Kyuubi community can only grow with the combined efforts of every user and developer.
Apache Kyuubi 社区的发展离不开每一位用户和开发者的共同努力。

The Apache Kyuubi community appreciates every user and developer who uses and cares about Kyuubi. We are committed to listening to each one of you to make the community more prosperous.
Apache Kyuubi 社区感激每一位使用和关注 Kyuubi 的用户和开发者，我们努力倾听每一个人的声音，让社区变得更加繁荣。

So, the starting point for this topic is to encourage you, as a user or developer, to leave your real-world scenarios so that we can subsequently explore new and better solutions, schedule the releases, and solve real-world problems based on these scenarios!
所以，这个议题的出发点是希望作为用户或者开发者的您留下您的实际使用场景，以便我们后续基于此探索更好的解决方案，制定版本计划，解决实际问题。

We look forward to your comments in the comments space, including but not limited to the following areas.
我们期待您能在评论区留言，其内容包括但不限于以下方面。

- Organization / 组织: e.g. NetEase Corp.
- Region / 地区:  e.g. Hangzhou, China
- Contact / 联系方式: e.g. yao@apache.org
- Usage Scenario / 使用场景: e.g., we use Apache Kyuubi as a multi-tenant Spark server providing Spark as a service, which facilitates our users to build a lot of big data workloads, including BI reporting, ETL, etc. It's a perfect tool to move Hive QL jobs smoothly to Spark SQL. 

Bests / 致敬,

Apache Kyuubi Community

**Comment by yaooqinn** [comment]
<img src="https://user-images.githubusercontent.com/8326978/129294930-fd55b761-1ff1-4cee-881d-b9a7fc1970f3.png" height="50%" width="50%" />

- Organization / 组织: NetEase Digital Sail.
- Region / 地区:  Hangzhou, China
- Contact / 联系方式: yao@apache.org
- Usage Scenario / 使用场景: Build Spark as a service for our users in an out-of-the-box way, running Kyuubi and Spark on both YARN and Kubernetes, both private Hadoop clusters and cloud services, like AWS. https://sf.163.com/

**Comment by servlet001** [comment]
![fcbox com](https://user-images.githubusercontent.com/8198379/129300401-d5ca1972-f533-41d0-82fb-fe4de9e44f75.png)

- Organization / 组织: 深圳丰巢科技
- Region / 地区: 中国 广东深圳
- Contact / 联系方式: lizhong@fcbox.com
- Usage Scenario / 使用场景: 基于kyuubi，在hue上进行SparkSQL交互式查询，beeline执行SparkSQL数仓ETL任务。[https://www.fcbox.com](https://www.fcbox.com/)

**Comment by hddong** [comment]
![cmss](https://user-images.githubusercontent.com/17537134/129313257-f1803638-aabe-457f-a9d4-24e9d575afb7.jpg)
* Organization / 组织: CMSS(CM Soft) / 中移（苏州）软件技术有限公司.
* Region / 地区:  Suzhou, China
* Contact / 联系方式: [hongdongdong_yewu@cmss.chinamobile.com](mailto:hongdongdong_yewu@cmss.chinamobile.com)
* Usage Scenario / 使用场景:  Provide customers with the ability to connect our LakeHouse in [ecloud](https://ecloud.10086.cn/home/product-introduction/lakehouse). And as multi-tenant Thrift JDBC/ODBC server in our private hadoop clusters.

**Comment by iodone** [comment]
![image](https://user-images.githubusercontent.com/5451385/129302947-d87d6cb0-d499-49db-9a34-78f76e7d3458.png)

- Organization / 组织: 小米大数据团队
- Region / 地区: 武汉, 中国
- Contact / 联系方式: zhangyaodong@xiaomi.com
- Usage Scenario/使用场景: OLAP 分析场景，基于Kyuubi 代理SparkSQL/Presto/Kylin等计算引擎，为用户提供一致易用的数据分析服务。

**Comment by pan3793** [comment]
<img src="https://user-images.githubusercontent.com/26535726/129300269-10c8f6c5-6293-4fb9-b90b-7175de6efaff.png" height="50%" width="50%" />

- Organization / 组织: 众言科技
- Region / 地区:  中国，上海
- Contact / 联系方式: bigdata@idiaoyan.com
- Usage Scenario / 使用场景: 使用 Kyuubi 与 CDH 6 集成，将 Spark SQL 暴露成与 HiveServer2 兼容的服务，提供与 BeeLine、HUE、Superset 的集成，用以支撑 ETL Batch 任务、OLAP 分析、Ad-Hoc 查询等场景，并扩展 Spark DataSource，实现 Hive、Iceberg、ClickHouse([Spark on ClickHouse](https://github.com/housepower/spark-clickhouse-connector))、MongoDB、MySQL 联邦查询。

**Comment by YangPeng007** [comment]
<img width="250" alt="image" src="https://user-images.githubusercontent.com/26535726/185356979-2c261f32-0ee5-4d68-92de-e48c47c6698d.png">

Organization / 组织: 网易传媒大数据技术中心
Region / 地区: 北京, 中国
Contact / 联系方式: pengyang1@corp.netease.com
Usage Scenario/使用场景: 基于Kyuubi实现了hive迁移spark，Spark On K8S平滑迁移，Ad-Hoc，Serverless Spark框架解决了 离线和在线业务 容器内混部问题

**Comment by runzhliu** [comment]
![image](https://user-images.githubusercontent.com/20874664/129315393-d6934c32-3cd9-4792-a54d-0c1d9e3742cb.png)

Organization / 组织: [Tencent Cloud Elastic MapReduce](https://cloud.tencent.com/product/emr)
Region / 地区: Shenzhen, China
Contact / 联系方式: runzhliu@tencent.com
Usage Scenario / 使用场景: Tencent Cloud Elastic MapReduce is an elastic and open-source cloud-based Hadoop service, Kyuubi is one of the provided services.

**Comment by lxk000000** [comment]
![WPS图片-修改尺寸](https://user-images.githubusercontent.com/26569836/129321146-ba609987-b53a-4b50-8dab-34175050825d.png)


Organization / 组织: T3出行
Region / 地区: 中国，南京
Contact / 联系方式: lixk000000@163.com
Usage Scenario / 使用场景: Kyuubi 在 T3 出行被应用于湖仓一体的架构之上。借助于 HiveServer2 并与 HUE、DolphinScheduler、达芬奇等 BI 工具的集成，支撑了部分场景的数据入湖(Hudi 格式)、离线业务的 ETL 以及大量的 AD Hoc、OLAP 场景。

**Comment by i7xh** [comment]
![zhihu_profile](https://user-images.githubusercontent.com/635169/129354832-92970119-ed56-4307-b5d0-b34d9b39681c.png)
Organization / 组织: Zhihu Inc.
Region / 地区: Peking, China
Contact / 联系方式: h@zhihu.com
Usage Scenario / 使用场景: Support Zhihu BigData Ad-Hoc Query / ETL Data Processing via multi-tenant SparkSQL ThriftService based on **Kyuubi** Framework

**Comment by gdc-exa** [comment]
![image](https://user-images.githubusercontent.com/89444188/130571101-bf57eb43-1cd6-4fb5-87c2-35e43f0ec81d.png)
Organization / 组织: 网易游戏
Region / 地区: 中国，广州
Contact / 联系方式: whke@corp.netease.com
Usage Scenario / 使用场景: 网易游戏大数据平台基于内部的统一 SQL 框架 GDCSQL，使用 Kyuubi 实现 Spark 引擎多租户的支持，支撑 Adhoc 以及部分 ETL 业务。

**Comment by 942011334** [comment]
![bilibili](https://user-images.githubusercontent.com/26920583/133870642-4f6f9550-5e9b-49f0-8b18-e7330b864b1a.jpeg)
Organization / 组织: 哔哩哔哩
Region / 地区: 中国，上海
Contact / 联系方式: chenyukang@bilibili.com ; jiadongdong@bilibili.com
Usage Scenario / 使用场景: 基于 Kyuubi 架构，研发了 queue 模式，实现了基于 queue 的多租户以及 Kyuubi 资源调度等功能，替代了Spark Thrift Server，支撑着大量 Adhoc 业务。

**Comment by wangfann** [comment]
![logo](https://user-images.githubusercontent.com/6930421/133871083-303c2ad1-0757-47e6-addb-aa0fa7ab6fa4.jpg)
Organization / 组织: 金山办公
Region / 地区: 中国，北京
Contact / 联系方式: wangfan6@wps.cn;yanghongqiang@wps.cn
Usage Scenario / 使用场景: 使用 Kyuubi 服务替代 Spark Thrift Server，支撑公司 Spark SQL 相关的 OLAP 分析，ad-hoc 查询等业务。

**Comment by CCweixiao** [comment]
![合合信息](https://user-images.githubusercontent.com/27730211/136743981-bcce1102-ab1a-408e-842d-af1090bd0bbb.jpeg)
![扫描](https://user-images.githubusercontent.com/27730211/136743998-0d978564-df20-4a08-b330-7909b077e890.png)  ![启信宝](https://user-images.githubusercontent.com/27730211/136744009-64a54f63-d6f1-4ae7-b976-b24ce7d6cbe7.png) ![名片](https://user-images.githubusercontent.com/27730211/136744016-afeba211-dfc1-44e5-9643-7ad56d4f9a91.png)


Organization / 组织: 合合信息
Region / 地区: 中国，上海
Contact / 联系方式: leo_jie@intsig.net
Usage Scenario / 使用场景: 将 Kyuubi、Spark 3.1 适配至 CDH5 (开启 Kerberos)，使用其多租户能力及引擎 USER、CONNECTION 级别共享等优良特性，以替代 Spark Thrift Server，支撑公司 Spark SQL 相关的 OLAP 分析，Adhoc 查询，以及 Tableau 中大量 STS 数据源刷新任务等业务。
参考文章链接：https://mp.weixin.qq.com/s/EyG9SfYatyGxgqrvqBbbdA

**Comment by ChengbingLiu** [comment]
<img src="https://user-images.githubusercontent.com/5031567/136747880-74ed47a0-0846-4987-8048-f6a26f5384b2.jpg" width="400">

Organization / 组织: iQIYI, Inc.
Region / 地区: Shanghai & Beijing, China
Contact / 联系方式: wangzhen07@qiyi.com; liuchengbing@qiyi.com
Usage Scenario / 使用场景: We use Kyuubi as a backend service for our unified SQL engine, Pilot, which runs a lot of scheduled and ad-hoc queries per day. Kyuubi is also used to facilitate the transition from Hive execution engine to Spark.

**Reply by turboFei** [reply]
<img src="https://user-images.githubusercontent.com/6757692/137426008-50aa37dd-c977-4ce9-a23f-5c2f909c445b.jpg " width="400">


Organization: eBay Inc.
Region: Shanghai, China & San Jose, California, America
Contact: fwang12@ebay.com
Usage Scenario: We have deployed Kyuubi on production. We plan to take the place of HiveServer2 with Kyuubi at first and then build unified Spark service based on Kyuubi in the future.

**Comment by bakea** [comment]
![image](https://user-images.githubusercontent.com/16595827/137583322-5dfdec03-6d7a-4cde-bc3f-33788318d47f.png)

Organization / 组织: 新华三大数据.
Region / 地区: 中国，杭州/郑州
Contact / 联系方式: shi.ningninga@h3c.com
Usage Scenario / 使用场景: 使用kyuubi做为Spark多租户解决方案，利用user、connection多级别灵活资源隔离方式，代替SparkThriftserver，解决原生STS的扩展性问题。

**Comment by zhenjiaguo** [comment]
<img src="https://user-images.githubusercontent.com/29809822/139858614-324cbae0-d1ce-450d-babc-7dd0b07ce3ef.jpeg" width=100% />

Organization / 组织: 海康威视.
Region / 地区: 中国，杭州
Contact / 联系方式: liuqinglin5@hikvision.com
Usage Scenario / 使用场景: 使用Apache Kyuubi替换原始的Spark Thrift Server，提供一种高并发、多租户、资源隔离的SparkSQL的执行能力，支撑大数据OLAP分析、ad-hoc查询等业务。

**Comment by jiaoqingbo** [comment]
<img width="590" alt="image" src="https://user-images.githubusercontent.com/26535726/185356461-35114817-2962-46a1-b39c-dd2231c6b74b.png">

Organization / 组织: 亚信科技
Region / 地区: 中国，北京
Contact / 联系方式: jiaoqb@asiainfo.com
Usage Scenario / 使用场景: 使用Kyuubi替代Spark Thrift Server，作为Spark多租户解决方案。

**Reply by yaooqinn** [reply]
Cool!

**Comment by turboFei** [comment]
<img src="https://user-images.githubusercontent.com/6757692/137426008-50aa37dd-c977-4ce9-a23f-5c2f909c445b.jpg " width="400">


Organization: eBay Inc.
Region: Shanghai, China & San Jose, California, America
Contact: fwang12@ebay.com
Usage Scenario: Building Unified & Serverless Spark with Kyuubi.
- Unified gateway for Spark, unique endpoint to serve multiple data centers
- Support to execute SQL query, Scala code and Spark batch application submission
- Take the place of HiveServer2 with Kyuubi

**Reply by turboFei** [reply]
Sharing link: [eBay Unified & ServerLess Spark Gateway on Kyuubi](https://www.bilibili.com/video/BV1oa411R7BC?is_story_h5=false&p=1&share_from=ugc&share_medium=iphone&share_plat=ios&share_session_id=82A8C26F-8516-495C-98C1-F1732B339124&share_source=COPY&share_tag=s_i&timestamp=1660814673&unique_k=huX61kK)

**Comment by Narcasserun** [comment]
Organization / 组织: 安元科技
Region / 地区: 中国，南京
Contact / 联系方式: Narcasserun@hotmail.com
Usage Scenario / 使用场景: Kyuubi 在 安元科技被应用于湖仓一体的架构之上。与DolphinScheduler、元数据网关等工具的集成，支撑了部分场景的数据入湖、离线业务的 ETL 以及大量的 ADHoc、OLAP场景。

**Comment by cxzl25** [comment]
![c_logo2020](https://user-images.githubusercontent.com/3898450/145407876-bc86ea15-f17b-4dcf-9a02-686e84d9db36.png)

Organization / 组织: 携程
Region / 地区: 中国，上海
Contact / 联系方式: sychen@trip.com
Usage Scenario / 使用场景: 使用Kyuubi1.4.0+Spark3.2.0替代原来在Spark2实现的多租户Thrift Server，目前用于Adhoc查询，后续会灰度替换Spark2 Thrift Server，支持OLAP报表等场景。

**Comment by fuhao009** [comment]
Organization / 组织: 暂无
Region / 地区: 杭州, 中国
Contact / 联系方式: 15990155931@gmail.com
Usage Scenario/使用场景: 准备基于Kyuubi实现集成greenplum和tidb数据源查询

**Comment by yaroslavambrozyak** [comment]
![image](https://user-images.githubusercontent.com/26353169/150510739-425a644a-ad01-4f66-92c6-704c492a916d.png)

Organization: Womply
Region: San francisco, California, America
Contact: mmuthusamy@womply.com
Usage Scenario: We plan to use Kyuubi for our Anaytics workloads use cases such as business analysis and reporting.

**Comment by yangrong688** [comment]
![lingxing](https://user-images.githubusercontent.com/70557521/159272296-730671ba-1278-4055-8add-98f6d9670a20.png)

Organization / 组织: [LingXing](https://www.lingxing.com/)
Region / 地区:  ShenZhen, China
Contact / 联系方式: [yangrong.jxufe@gmail.com](mailto:yangrong.jxufe@gmail.com)
Usage Scenario / 使用场景: We use Kyuubi for batch processing senario and interact with Spark and YARN.

**Comment by dongkelun** [comment]
![ca1349540923dd54564ec454e258a4de9c82d158e2f4](https://user-images.githubusercontent.com/13376555/177115748-bbe40985-7034-4b30-a940-27d81f6cf42f.jpg)

Organization / 组织: [inspur cloud](https://cloud.inspur.com/)
Region / 地区: jinan, China
Contact / 联系方式: [dongkelun01@inspur.com](dongkelun01@inspur.com)
Usage Scenario / 使用场景: Spark Thrift Server HA.

**Reply by yaooqinn** [reply]
thanks!

**Comment by cfmcgrady** [comment]
<img src="https://user-images.githubusercontent.com/8537877/177677575-f156f5e2-3dc5-4b0e-a00e-a5c35bc339f7.png" height="208px" width="332px" />

Organization / 组织: 丁香园大数据部门
Region / 地区: 中国，杭州
Contact / 联系方式: [fchen@apache.org](mailto:fchen@apache.org)
Usage Scenario / 使用场景: 利用Kyuubi的多租户/资源隔离/Spark扩展等特性，支撑公司的ETL，以及部分ad-hoc查询。

**Comment by bowenliang123** [comment]
![GF-Securities-Limited-Company-logo](https://user-images.githubusercontent.com/1935105/179951673-7c0451c9-281c-45b6-84a3-f701fa10d7b6.jpeg)


Organization / 组织:  [广发证券 Guangfa Securities](http://www.gf.com.cn/)
Region / 地区: 中国，广州
Contact / 联系方式: [liangbowen@gf.com.cn](mailto:liangbowen@gf.com.cn)
Usage Scenario / 使用场景: 使用 Kyuubi 1.6.0 + Spark3.3.0 + Iceberg0.14 提供整合的数据服务能力。目前用于Adhoc查询和只读型计算检核任务。未来定制身份认证插件并结合Authz插件，对接Ranger2.3.0提供用户粒度及行权限过滤的的数据查询加工能力，并推广到ETL加工、支持OLAP分析等场景。

**Comment by magicsilence** [comment]
![972a5c2d-a956-441e-bb31-c77548101b9a-106x106](https://user-images.githubusercontent.com/823374/179950723-0d08a453-df3c-4e2b-b4b4-e1324c48636f.png)

Organization / 组织: fordeal
Region / 地区: 中国，广州
Contact / 联系方式: [yundian@fordeal.com](mailto:yundian@fordeal.com)
Usage Scenario / 使用场景:  替代sts,  与dolphinScheduler，redash 集成，主要用spark on k8s场景下的任务隔离、资源共享等功能。服务数据开发、adhoc场景。

**Reply by yaooqinn** [reply]
> 与dolphinScheduler，redash 集成。

有机会可以分享下集成的经验。完善一下社区的文档建设

**Reply by magicsilence** [reply]
二开比较多，有机会补一个文档链接，详细说明

**Comment by ITSKATYMABYBABY** [comment]


**Reply by yaooqinn** [reply]
Hi @ITSKATYMABYBABY, here are the steps for joining the lists https://kyuubi.apache.org/mailing_lists.html

**Comment by lukeyan2023** [comment]
![微信图片_20220810095447](https://user-images.githubusercontent.com/108530647/183793354-e0fc5200-da91-4062-a120-f2dee1e16c77.jpg)
Organization / 组织: 中企通信 数据科学与创新团队
Region / 地区: 中国，广州/上海/成都
Contact / 联系方式: luke.yan@china-entercom.net
Usage Scenario / 使用场景: 使用 Kyuubi 和 Spark3 替换 Spark2 ，支持ETL 和 Adhoc 查询任务。正在基于 Kyuubi +Spark/Flink + Iceberg 进行数据湖平台的构建。

**Comment by lianneli** [comment]
<img width="655" alt="B端业务-横版-蓝色" src="https://user-images.githubusercontent.com/19421187/184073008-a9dbeea5-db2e-45ac-95a8-5fc59fa6b793.png">

- Organization / 组织: DMALL  
- Region / 地区: Beijing, China
- Contact / 联系方式: gang.huang@dmall.com, ming.li2@dmall.com
- Usage Scenario / 使用场景: DMALL provides users with more customized Ad-hoc service with the help of Kyuubi's multi-tenant design. Users can use their own resources and tuning settings according to different usage scenarios, which greatly improves the overall query efficiency. Migrating the original Hive query engine to Kyuubi is on the way, for improving query efficiency. Meanwhile, it also meets lots of scenarios such as complex data types, interactive UDF verification and ETL pre exploration. Now, Kyuubi runs stably on YARN, and Kyuubi on K8s is under test, which will be comprehensively promoted in the future.

**Comment by ConstanceLiang** [comment]
<img width="437" alt="image" src="https://user-images.githubusercontent.com/26535726/185527988-f27291e2-0641-4623-b6c6-261ca8509e28.png">

Organization / 组织:  北京万方数据股份有限公司
Region / 地区:  北京
Contact / 联系方式: xuxinhao@wanfangdata.com
Usage Scenario / 使用场景：跨数据源关联查询，对接metabase实现BI

**Comment by ConstanceLiang** [comment]
![177286467_1_q2_16608144898HMJHD (1)](https://user-images.githubusercontent.com/107614867/185527770-1a09259b-338f-4235-863f-e443e8b9e009.png)

Organization / 组织: 华泰证券
Region / 地区:  Nanjing, China
Contact / 联系方式:  hexiaoming@htsc.com
Usage Scenario / 使用场景：作为spark连接池使用，解决spark应用启动时反复启动完成性能资源浪费。

**Comment by ConstanceLiang** [comment]
<img width="180" alt="177286467_2_q2_1660815886AdzJ2j" src="https://user-images.githubusercontent.com/107614867/185528217-a21e447d-1bb5-4504-804a-9a0c247b7ad8.png">

Organization / 组织: 凯捷中国咨询 
Region / 地区: shanghai，china 
Contact / 联系方式: izonda@163.com 
Usage Scenario / 使用场景：ad-hoc场景的应用 spark多版本 云上环境

**Comment by ConstanceLiang** [comment]
![177286467_5_q2_1660875567Gi23md](https://user-images.githubusercontent.com/107614867/185553209-d4e08274-f2ea-4f16-9db9-c87a0f0eaded.png)


Organization / 组织:  协创数据
Region / 地区:  合肥
Contact / 联系方式：664283422@qq.com
Usage Scenario / 使用场景：多租户，身份校验，可以帮助不通项目组做到资源和数据隔离

**Comment by ConstanceLiang** [comment]
![177286467_7_q2_1660890415a7BxfJ](https://user-images.githubusercontent.com/107614867/185557838-f2283666-70a6-468e-b911-3bc2ce836a9b.png)


Organization / 组织：珠海必要工业科技有限公司
Region / 地区：中国北京市
Contact / 联系方式： renwendong@idstaff.com
Usage Scenario / 使用场景：数仓对外数据服务

**Comment by ConstanceLiang** [comment]
![177286467_8_q2_1660890874r38wiC](https://user-images.githubusercontent.com/107614867/185561133-a13ac0a8-f5f7-456b-aec5-25580213766d.png)


Organization / 组织：丝域科技
Region / 地区： 中国广州
Contact / 联系方式：chenhaifeng@hairology.cn
Usage Scenario / 使用场景：
1、替代SparkThrift。
2、使用Kyuubi多租户资源隔离的功能，避免多个用户使用导致进程卡死的情况。
3、使用Kyuubi的高可用功能，防止单节点故障

**Comment by ConstanceLiang** [comment]
![177286467_11_q2_166089217647FnZH (1)](https://user-images.githubusercontent.com/107614867/185563209-fcd95d7d-8ab5-4681-a8bd-0ac1620e8558.png)


Organization / 组织：汇付数据
Region / 地区： 上海，中国
Contact / 联系方式： shuai.chen@huifu.com
Usage Scenario / 使用场景：serverless sql gateway for lakehouse

**Comment by ConstanceLiang** [comment]
<img width="150" alt="177286467_12_q2_1661140627xGNK2n" src="https://user-images.githubusercontent.com/107614867/185848885-f9e0b277-6cf5-443a-9d94-41dc06098219.png">

Organization / 组织: 税友软件集团
Region / 地区:  中国杭州
Contact / 联系方式：wsk@servyou.com.cn
Usage Scenario / 使用场景：租户隔离的临时查询

**Comment by weipengffk** [comment]
Organization / 组织: 立创商城
Region / 地区: 中国-深圳
Contact / 联系方式:1310503090@qq.com
Usage Scenario / 使用场景: 使用dbeaver+kyuubi 给数据分析师&数据开发人员进行数据分析.  使用kyuubi 作为小部分ETL场景，湖仓一体方案预研

**Comment by minyk** [comment]
![nexr-logo](https://user-images.githubusercontent.com/1802676/190089491-9c92d3cc-dcc4-44f3-8894-f6e690e4f4b7.png)

Organization / 组织: kt NexR https://ktnexr.com (currently korean only)
   Region / 地区: Seoul, Republic of Korea
   Contact / 联系方式: drake.min@nexr.com, ebreeze@nexr.com
   Usage Scenario / 使用场景: The NexR Data Analytics Platform (NDAP) uses Apache Kyuubi as the JDBC frontend for Spark. Apache Kyuubi ensures enhanced support for customers, catering ad-hoc and/or batch query processing.

**Reply by minyk** [reply]
Unfortunately, kt NexR is no longer operating as of 12/26/2024.

**Comment by zengfan-houzz** [comment]
![image](https://user-images.githubusercontent.com/54294465/190507223-5f0b6655-ed19-457d-95a5-dd6e9d566bca.png)

Organization / 组织: Houzz
Region / 地区: Palo Alto, California, US
Contact / 联系方式: zeng@houzz.com
Usage Scenario / 使用场景: we use Apache Kyuubi as a multi-tenant Spark server providing SparkSQL service to our analysts, thus they can analyze the data locally using DBeaver and write their BI reports on Redash/Tableau.

**Comment by zhangxiang17** [comment]
<img width="198" alt="image" src="https://user-images.githubusercontent.com/20200008/211248097-86542fbc-c0b8-4461-92aa-dcf558ebc46b.png">

Organization / 组织: 中国电子云
Region / 地区: 中国
Usage Scenario / 使用场景: 由于其在多租、多级Share Level的特性， Kyuubi 作为SparkSQL 的服务网关在我们这里发挥特别重要的作用。

**Comment by Kiss736921** [comment]
<img width="493" alt="image" src="https://user-images.githubusercontent.com/43542750/225010993-88881d6a-0de1-41b6-a4d1-59b816ab0022.png">

Organization / 组织: BOSS
Region / 地区: Beijing, China
Contact / 联系方式: zoulimin@kanzhun.com
Usage Scenario / 使用场景: we use Apache Kyuubi as a multi-tenant SparkSQL server supports offline and ad-hoc query.

**Comment by lukaizj** [comment]
![企查查](https://github.com/apache/kyuubi/assets/92360203/40ad481b-d9b9-4ee2-9f36-b6ad033ce113)

Organization / 组织: 企查查
Corp.Region / 地区: 中国.苏州
Contact / 联系方式: lukai@greatld.com
Usage Scenario / 使用场景: 
1、使用kyuubi+spark3代替SparkThrift
2、数仓对外数据查询服务
3、使用tidb+flink+iceberg+kyuubi+hms湖仓一体方案预研，已小规模使用

**Comment by etheleon** [comment]
<img src="https://github.com/apache/kyuubi/assets/2868858/b069483f-aec2-4fa3-be7a-cb10c9fde550" width="100">

Organization / 组织: liftoff.io
Corp.Region / 地区: China.Beijing / Singapore 
Contact / 联系方式: [wgoi@liftoff.io](mailto:wgoi@liftoff.io)
Usage Scenario / 使用场景:
1、使用kyuubi+spark3代替SparkThrift

**Comment by kissycn** [comment]
![image](https://github.com/apache/kyuubi/assets/89451328/f3a7879c-9cdb-4bec-875d-a2cbc8f5d699)

Organization / 组织: 成都织数科技（dtweave.com）
Corp.Region / 地区: China.Chengdu
Contact / 联系方式: [kissycn@gmail.com](mailto:kissycn@gmail.com)
Usage Scenario / 使用场景: SQL Gateway

**Comment by SparksFyz** [comment]
![fe-platform_ac25fc3558e83d75edf4b36b8a7739e1f63f2ad6 svg](https://github.com/apache/kyuubi/assets/8748814/bc4deaf4-f226-4386-b2ce-7b0ad438f7ae)

Organization / 组织: 小红书
Region / 地区: Shanghai China
Contact / 联系方式: [yfan2@xiaohongshu.com](mailto:yfan2@xiaohongshu.com)
Usage Scenario / 使用场景: SQL Gateway，支撑Adhoc和ETL场景中的Spark/Hive/Presto/Starrocks等计算引擎

**Comment by thijs-s** [comment]
![ilum-logo2-2](https://github.com/user-attachments/assets/6c14fc7f-a559-4142-a284-33562766de0d)

Organization / 组织: [Ilum Labs LLC](http://ilum.cloud/)
Region / 地区: USA
Contact / 联系方式: info@ilum.cloud
Usage Scenario / 使用场景: We use Apache Kyuubi for our free data lakehouse solution as an sql gateway

**Comment by ASiegeLion** [comment]
![image](https://github.com/user-attachments/assets/c9036042-997c-4446-acf9-226141697b31)
Organization / 组织: YY( https://www.yy.com)
Region / 地区: GuangZhou, China
Contact / 联系方式: https://www.yy.com/web/jinhong/#/connect
Usage Scenario / 使用场景: 作为flink,trino,spark引擎的统一sql网关，承接用户的etl,olap 等sql任务

**Comment by HYBG-1126** [comment]
![6e60d976-aab1-4d1c-9bf9-0c6cf82f2961](https://github.com/user-attachments/assets/2e65d688-14c9-4309-a78b-f00c824e8df2)
Organization / 组织: 竞技世界( jjworld)
Region / 地区: Beijing, China
Contact / 联系方式: https://www.jj.cn
Usage Scenario / 使用场景: 作为Spark引擎的统一sql网关，替代hive，hue中提升开发效率，离线数仓中提升任务运行效率

**Comment by gckalia** [comment]
I have the following scenario. I have a number of applications who would like to independently use SQL queries using spark. The data is ingested by independent plugins, which write partitions as parquet on HDFS. Each application requires only a subset of the partition for its queries. So I would like to have separate sparksession where the tables are loaded using only the relevant partitions that an application requires. So when the application opens a connection it should be able to load and access the tables that are behind the spark session. Could you explain how the clients can specify which data partitions are required to be loaded into a view in the spark session and hence send query requests for that session. What is the lifecycle of such a session, is this session active until the connection is active?

**Comment by Ignalina** [comment]
<img width="256" height="281" alt="image" src="https://github.com/user-attachments/assets/4f6c8eba-22dd-4c87-91ed-d11002a0be7a" />


Organization / 组织: [X14 kod & infrastruktur AB](http://www.x14.se/)
Region / 地区: Sweden,Europe
Contact / 联系方式: [gustav@x14.se](mailto:gustav@x14.se)
Usage Scenario / 使用场景: We use Apache Kyuubi for our on prem data lakehouse applience  sql gateway
